nucleus/admin/rest/rest-service/src/main/java/org/glassfish/admin/rest/logviewer/LogRecord.java

```diff
@@ -17,44 +17,48 @@
 
 package org.glassfish.admin.rest.logviewer;
 
-import java.io.StringWriter;
+import java.io.Serializable;
 import java.time.OffsetDateTime;
-import java.util.Date;
-
-import javax.xml.parsers.DocumentBuilder;
-import javax.xml.parsers.DocumentBuilderFactory;
-import javax.xml.parsers.ParserConfigurationException;
-import javax.xml.transform.OutputKeys;
-import javax.xml.transform.Result;
-import javax.xml.transform.Source;
-import javax.xml.transform.Transformer;
-import javax.xml.transform.TransformerConfigurationException;
-import javax.xml.transform.TransformerException;
-import javax.xml.transform.TransformerFactory;
-import javax.xml.transform.dom.DOMSource;
-import javax.xml.transform.stream.StreamResult;
+import java.util.List;
+
+import javax.xml.stream.XMLStreamException;
+import javax.xml.stream.XMLStreamWriter;
 
 import org.codehaus.jettison.json.JSONException;
 import org.codehaus.jettison.json.JSONObject;
-import org.w3c.dom.Document;
-import org.w3c.dom.Element;
-import org.w3c.dom.Node;
 
 /**
- * internal REST wrapper for a log record will be used to emit JSON easily with Jackson framework
+ * Internal REST wrapper for a log record will be used to emit JSON easily
+ * with Jackson framework.
  *
  * @author ludo
  */
+@SuppressWarnings("unused")
```

Comment:
I don't like these annotations - why it is here?
If there is some unused field, it should be removed OR at least documented why is it here.